compiler/x/codegen/OMRCodeGenerator.cpp

```diff
@@ -171,7 +171,18 @@ void TR_X86ProcessorInfo::initialize(bool force)
             uint32_t extended_model = getCPUModel(_processorSignature) + (getCPUExtendedModel(_processorSignature) << 4);
             switch (extended_model)
                {
-               case 0x55:
+               case 0xcf:
+                  _processorDescription |= TR_ProcessorIntelEmeraldRapids; break;
+               case 0x8f:
+                  _processorDescription |= TR_ProcessorIntelSapphireRapids; break;
+               case 0x6a:  // IceLake_X
+               case 0x6c:  // IceLake_D
+               case 0x7d:  // IceLake
```

Comment:
I could not find ```7D``` documented. I assume its a client processor?

For the client ice lake processors I see

|Processor|Family|Extended Family|Model|
|--|--|--|--|
|Ice Lake Client (U) |0x6|0x7|0xe|
|Ice Lake Client (Y) |0x6|0x7|0xe|

Reply:
Yes, see Vol 4., Table 2-1